scaffold/make/setup.mk

```diff
@@ -38,6 +38,7 @@ _clean:
 	rm -rf ./tests/cypress 2>&1
 	rm -rf ./tests/phpunit 2>&1
 	rm -rf ./vendor 2>&1
+	rm -rf decoupled monolithic
```

Comment:
Don't run several removals with only command, separate this in 2 different `rm` instructions. On the other hand, is a little confusing this naming at the final scaffolded files, the structure should be the same regardless the kind of project.